netty-server/src/main/scala/org/http4s/netty/NettyBuilder.scala

```diff
@@ -254,64 +272,73 @@ class MikuBuilder[F[_]](
   /** A stream transformation that registers our channels and
```

Comment:
It's not really a stream transformation anymore.  It's just an effectful function.

Reply:
fair. I'll make that distinction